docker/test

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -ex
+HERE=$(dirname $0)
+. $HERE/common
+
+NAME_SERVER=orderly_server
+
+function cleanup {
+  echo "Cleaning up"
+  docker kill $NAME_SERVER > /dev/null || true
+  rm -rf $PACKAGE_ROOT/orderly-demo
+}
+
+trap cleanup EXIT
+
+rm -rf orderly-demo
+git clone https://github.com/vimc/orderly-demo
+Rscript -e 'setwd("orderly-demo")' -e 'orderly::orderly_rebuild()'
```

Comment:
Looks like it's just empty directories being created, so try this:

```suggestion
mkdir -p orderly-demo/runner/log
mkdir -p orderly-demo/runner/id
Rscript -e 'setwd("orderly-demo")' -e 'orderly::orderly_rebuild()'
```

that will ensure that they exist before the container starts and so they'll be owned by the script user and can be deleted.

The alternative way would be to bring up an docker container without an orderly archive but using a go signal. Then exec into that container and clone the orderly-demo repo and set the go signal - that's faffier but if we needed to actually run something that'd be needed